lib/bugsnag/helpers.rb

```diff
@@ -1,3 +1,4 @@
+require 'bugsnag/helpers/exceptions'
 require 'uri'
 require 'set'
 require 'json'
```

Comment:
I would ordinarily expect to see external requires followed by internal requires.